### PR TITLE
postcss-gap-properties: fix detection of `display: grid`

### PR DESCRIPTION
**Pull request by romainmenke**
fixes: https://github.com/csstools/postcss-plugins/issues/540

### CODE REVIEW

File: plugins/postcss-gap-properties/src/index.js [hunk 1]
```diff
@@ -17,6 +17,10 @@ function creator(opts) {
 			}
 
 			const isNotDisplayGrid = !(decl.parent.some((node) => {
+				if (node.type !== 'decl') {
```
Comment (romainmenke) [inline plugins/postcss-gap-properties/src/index.js:20]: When this is a `rule` or `at-rule` `prop` and `value` won't exist

Comment (iain-s-h) [reply plugins/postcss-gap-properties/src/index.js:20]: With this change I'm getting a runtime error on line 32 because node.prop is undefined.

Comment (romainmenke) [reply plugins/postcss-gap-properties/src/index.js:20]: thank you @iain-s-h !

**Review by Antonio-Laguna — APPROVED** [review]
LGTM!

**Pull request merged**